src/contexts/Prices/PricesProvider.tsx

```diff
@@ -83,16 +82,12 @@ const PricesProvider: React.FC = ({ children }) => {
       .catch((error) => console.log(error))
   }, [])
 
+  // TODO: Remove this logic. Replace it with coingecko sourced data once available.
   useEffect(() => {
-    const coingeckoIndexPriceUrl = `https://api.coingecko.com/api/v3/simple/token_price/ethereum?contract_addresses=${indexTokenAddress}&vs_currencies=usd`
```

Comment:
`token_price/ethereum` probably needs to switch to `token_price/polygon-pos` in these situations

Reply:
I misunderstood, you're correct